doc/bu-electrum-integration.md

```diff
@@ -1,31 +1,31 @@
-Electrum Server Basic integration.
-==================================
+# Electrum Server
 
-Starting from version 1.6  BUcash it's possible to build a rust electrum server implementation binary (electrs)
-and spawn the process during the startup phase. Then you could use this electum server
-along with the electron cash wallet.
+## Overview
 
-To build the [electrs](https://github.com/dagurval/electrs) binaries use:
+Bitcoin Unlimited includes support for electrum server (from version 1.6, Linux x64 only).
 
-```sh
-make electrs
-```
+The server is maintained as a [separate piece of software](https://github.com/bitcoinunlimited/electrscash), but is integrated with the node software. Its process has the same lifetime as the bitcoind process, uses the same log files and adds functionality to the same RPC interface.
```

Comment:
this is a real nit, but I'd rather use https://github.com/BitcoinUnlimited/ElectrsCash rather than all lowecase URL :laughing: